internal/reconciliations/gateway/dns_entry_test.go

```diff
@@ -84,8 +85,8 @@ var _ = Describe("DNSEntry", func() {
 func getTestIstioIngressGatewayDnsBasedService() corev1.Service {
 	return corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "istio-ingressgateway",
-			Namespace: "istio-system",
+			Name:      KymaGatewayName,
```

Comment:
Needs to be updated as well, that's why I personally consider re-using implementation vars for tests a bad practice.

Reply:
Some kind of wrong replace I guess. No idea how/why.